tests/performance/shatal/internal/shatal/creator.go

```diff
@@ -109,17 +110,17 @@ func (s *Creator) createVMs(ctx context.Context) {
 				},
 			},
 			Spec: v1alpha2.VirtualMachineSpec{
+				VirtualMachineClassName:  "generic-v1",
 				RunPolicy:                v1alpha2.AlwaysOnPolicy,
 				EnableParavirtualization: true,
 				OsType:                   v1alpha2.GenericOs,
 				Bootloader:               "BIOS",
 				CPU: v1alpha2.CPUSpec{
-					VirtualMachineCPUModel: "generic-v1",
-					Cores:                  1,
-					CoreFraction:           "10%",
+					Cores:        1,
+					CoreFraction: "10%",
 				},
 				Memory: v1alpha2.MemorySpec{
-					Size: "500Mi",
+					Size: resource.MustParse("512Mi"),
```

Comment:
Why has the size changed here?

Reply:
We used to specified the memory size as a string, but in new versions I replace this field with resources.Quantity.
500 Ml looks strange to me.